src/Main.java

```diff
@@ -1,41 +1,38 @@
-import java.util.Arrays;
-import java.util.Scanner;
+import java.util.*;
+//  I did NOT create this myself. I create it by finding a parts in google and split it together...
 
 public class Main {
```

Comment:
It would be better to write different homeworks in different classes but not rewrite a task in one class. And better a is named your class according with a content of a class. Do not call all classes a Main